.gulp/typescript.iced

```diff
@@ -11,6 +11,9 @@ copyDtsFiles = (done) =>
     .on 'end', () => 
     source ["#{basefolder}/src/autorest-core/dist/**/*.d.ts","!#{basefolder}/src/autorest-core/dist/test/**" ]
       .pipe destination "#{basefolder}/src/autorest/dist/lib/core"
+    .on 'end', () =>
+     source ["#{basefolder}/src/autorest/vscode/**/*.d.ts" ]
```

Comment:
I copied the vscode interfaces out of the four seperate packages that they come in. I only need the d.ts.files and didn't need the actual code.